modules/oe_content_organisation/oe_content_organisation.post_update.php

```diff
@@ -51,6 +56,11 @@ function oe_content_organisation_post_update_00004(): void {
       ->updateFromStorageRecord($form_display, $form_display_values);
     $updated_form_display->save();
   }
+
+  // Install submodule to allow referencing Person content types.
+  \Drupal::service('module_installer')->install([
+    'oe_content_organisation_person',
+  ]);
```

Comment:
Since the main module doesn't depend from `oe_content_organisation_person` (because it cannot, or else we get the circular dependency), we should remove this from here, or else and update state would different from and install state.